.travis.yml

```diff
@@ -5,3 +5,6 @@ before_script:
   - bundle exec thor cldr:download
 script:
   - bundle exec ruby test/all.rb
+
+before_install:
+  - gem install bundler
```

Comment:
Is this necessary because of incompatibilities with bundler v2.0?

Reply:
Ok, just ran into something similar. I had to add `gem update --system` to .travis.yml to fix master. Might want to add that to your branch as well (i.e. merge master).

Reply:
Sweet :)